src/lib.rs

```diff
@@ -272,7 +269,7 @@ impl Pty {
   /// See the docstring of the class for an usage example.
   #[napi]
   #[allow(dead_code)]
-  pub fn fd(&mut self) -> Result<c_int, napi::Error> {
+  pub fn fd(&mut self) -> Result<RawFd, napi::Error> {
```

Comment:
```suggestion
  pub fn fd(&mut self) -> Result<i32, napi::Error> {
```

it's disgusting, but i think we need to cast this to `i32` so that the codegen emits a nice `number` https://napi.rs/docs/concepts/function